mreg/models.py

```diff
@@ -548,6 +549,16 @@ def get_first_unused(self):
                 return str(ip)
         return None
 
+    def get_random_unused(self):
+        """
+        Return a random unused IP, if any.
+        """
+
+        unused = self.get_unused_ipaddresses()
+        if unused:
+            return str(random.choice(tuple(unused)))
```

Comment:
Not random enough for ipv6, as `get_unused_ipaddresses()` will only return the first 4000 addresses. A normal ipv6 network has 2**64 ipaddresses and the random function should use the entire address space.
https://github.com/unioslo/mreg/blob/3a595a8d67fc08ee17b9b3bc810b94d7913445ce/mreg/models.py#L514-L523